gulpfile.js

```diff
@@ -52,6 +52,50 @@ gulp.task('checkNativeDependencies', (done) => {
     }
     done();
 });
+gulp.task('checkNpmDependencies', (done) => {
```

Comment:
What's this for? Seems odd. Why do we need to check these? Did you force change the package-lock again?

Reply:
Yes, and wrote a script to ensure it doesn't regress.
Else we need to update a tonne of dependencies.
I'd rather create a seprate issue to update all packages to latest versions than do that now.

Reply:
Could you add a comment as to why you added this into the gulpfile?

Reply:
Sure

Reply:
Done